packages/ketcher-core/src/application/render/renderers/BaseRenderer.ts

```diff
@@ -83,6 +85,10 @@ export abstract class BaseRenderer implements IBaseRenderer {
     return this.rootBBox?.y || 0;
   }
 
+  public get selectionPoints(): Vec2[] | undefined {
+    return undefined;
```

Comment:
I usually prefer having `null` as a explicit absence of value and `undefined` as non-explicit, but it's okay